ports/nrf/drivers/usb/usb_cdc.c

```diff
@@ -155,12 +155,13 @@ static void cdc_task(bool tx)
     }
 }
 
-static void usb_cdc_loop(void) {
+void usb_cdc_loop(void) {
     tud_task();
     cdc_task(true);
 }
 
 void tud_cdc_rx_cb(uint8_t itf) {
+    tud_task();
```

Comment:
Is this correct, to call `tud_task()` from within this callback?  Isn't this callback already being called from TinyUSB?  So you are calling back into TinyUSB and that may lead to reentrancy issues.

Reply:
I don't understand it either, but it doesn't work unless this is called here.

Reply:
I'll retest this without the call.

Reply:
Okay I think this has nothing to do with TinyUSB, it's maybe unaligned access or something like that. Please replace this:
```c
            int c;
            uint32_t count = tud_cdc_read(&c, 1);
            (void)count;
```

With this

```c
            int c = tud_cdc_read_char();
```

And remove the `tud_task()` call from `tud_cdc_rx_cb`. I tested that change and I can interrupt scripts just fine.

Reply:
I have updated the PR. It's better if you cross-check it.

Reply:
Tested again and it's working fine.